internal/lsp/server.go

```diff
@@ -591,12 +602,20 @@ func (s *Server) handleInitialized(ctx context.Context, req *lsproto.RequestMess
 		s.projectService.SetCompilerOptionsForInferredProjects(s.compilerOptionsForInferredProjects)
 	}
 
+	if s.initializeParams.RootUri.Value != "" {
+		s.projectService.Workspace.SetRoot(ls.DocumentURIToFileName(s.initializeParams.RootUri.Value))
+	}
+	if s.initializeParams.WorkspaceFolders != nil {
+		for _, folder := range s.initializeParams.WorkspaceFolders.Value {
+			s.projectService.Workspace.AddFolder(ls.DocumentURIToFileName(lsproto.DocumentUri(folder.Uri)))
```

Comment:
Do you know why we need to handle this kind of thing? Typically this stuff is only ever used if we plan to explicitly handle multi-root workspaces, but I don't think we have a reason to do that, since we're great at just accepting whatever gets opened anyway?